integration-tests/verification-flow/sales-demo-auto-loan.test.ts

```diff
@@ -0,0 +1,357 @@
+    type: 'JsonSchemaValidator2018',
+    parsingOptions: {
+      useDefaults: true,
+      defaultMinimumInteger: -4294967295,
+      defaultMinimumDate: -17592186044415,
+      defaultDecimalPlaces: 4,
+    },
+    version: '0.3.0',
+  },
+  name: 'ForSurBiometric',
+  cryptoVersion: '0.5.0',
+  proof: {
+    '@context': [
+      {
+        sec: 'https://w3id.org/security#',
+        proof: {
+          '@id': 'sec:proof',
+          '@type': '@id',
+          '@container': '@graph',
+        },
+      },
+      'https://ld.dock.io/security/bbs/v1',
+    ],
+    type: 'Bls12381BBS+SignatureDock2022',
+    created: '2024-06-11T19:52:42Z',
+    verificationMethod:
+      'did:dock:5HLbQLSmirNuZVRsdWKbsgdajw9QTGzSFJABSVzMT5EBj5sb#keys-2',
+    proofPurpose: 'assertionMethod',
+    proofValue:
+      'zY6FZ8hvC7rcJFBdCzDEnT3RUxeF4kjPjD3e6Eyg8wnqs9cPuvXaZ9Vcwke895LqCUNfAbUhZZDiMMb6FMLNrnNYQv2ikrjzpcU9gVuVaCZQH8rQvGNqXMBP5Cf3wjN7VFuQEp75Fp9sAkDfkSswQcNzut',
+  },
+};
+
+const credential2 = {
+  '@context': [
+    'https://www.w3.org/2018/credentials/v1',
+    'https://w3id.org/vc/status-list/2021/v1',
+    {
+      dk: 'https://ld.dock.io/credentials#',
+      EquiNetCreditScore: 'dk:EquiNetCreditScore',
+      credit_score: 'dk:credit_score',
+      name: 'dk:name',
+      description: 'dk:description',
+      logo: 'dk:logo',
+    },
+  ],
+  credentialStatus: {
+    id: 'status-list2021:dock:0x1253ed4ab4a818aaa6609a21f932dca0f0e58430d2b19efd1a008bac8e98a353#38',
+    type: 'StatusList2021Entry',
+    statusListIndex: '38',
+    statusListCredential:
+      'status-list2021:dock:0x1253ed4ab4a818aaa6609a21f932dca0f0e58430d2b19efd1a008bac8e98a353',
+    statusPurpose: 'suspension',
+  },
+  id: 'https://creds-testnet.dock.io/9ee9d7581666c6a05a68a3cf730be4e8648d7ef74198aa8d63ecad624940eb38',
+  type: ['VerifiableCredential', 'EquiNetCreditScore'],
+  credentialSubject: {
+    id: 'did:key:z6MkoWAL66HUG7SHmJpwisjbUrjgaUZZnesNXp5m2CDmrFkT',
+    credit_score: 707,
+  },
+  issuanceDate: '2024-06-11T19:37:10.792Z',
+  name: 'EquiNet - Credit Score',
+  description:
+    'This schema represents a Verified Credit Score Credential, issued by EquiNet. It standardizes the presentation of credit scores for reliable and efficient verification processes.',
+  issuer: {
+    name: 'EquiNET',
+    description: 'EquiNet is the credit bureau.',
+    logo: 'https://img.dock.io/9f327cafda3be5f0cff0da2df44c55da',
+    id: 'did:dock:5CKsfvaE68mvRhdn3dDXG4KpWzuvaUNdBbiu6sFUuPK9rw66',
+  },
+  proof: {
+    type: 'Ed25519Signature2018',
+    created: '2024-06-11T19:37:11Z',
+    verificationMethod:
+      'did:dock:5CKsfvaE68mvRhdn3dDXG4KpWzuvaUNdBbiu6sFUuPK9rw66#keys-1',
+    proofPurpose: 'assertionMethod',
+    jws: 'eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..o1pP7OiFZ_IeaLiebDIj6cl3-mor-61RoFLNc_HNyqG4WCfAkXBGiDZi5I3s50xsh3QjlMdFO3s7kswq4DwuCg',
+  },
+};
+
+const credential3 = {
```

Comment:
minor: descriptive names of the creds might be more helpful for understanding what part each plays

Reply:
there is a lint issue as well, let me fix that and improve the naming here